WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsWCPayNotSetup.swift

```diff
@@ -37,12 +37,12 @@ struct InPersonPaymentsWCPayNotSetup: View {
     }
 
     var setupURL: URL? {
-        ServiceLocator.stores.sessionManager.defaultSite?.adminURL(path: Constants.wcpaySetupPath)
-    }
-}
+        guard let adminURL = ServiceLocator.stores.sessionManager.defaultSite?.adminURL else {
```

Comment:
I'm having trouble hitting this use case. I have a site with WCPay active but not setup, and it hits this

https://github.com/woocommerce/woocommerce-ios/blob/81ed7b18078c58d3337f53a91d0f09c802bf3c3f/WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person%20Payments/CardPresentPaymentsOnboardingUseCase.swift#L144

before it gets to the `isWCPaySetupCompleted` that would trigger this flow.

I'll keep trying, but these changes look good and other than the concern about the project file noted below, ship it :)

Reply:
I ended up completing setup on this site and then forcing isWCPaySetupCompleted to return false. I then got the webview with the correct url.